Specs/Data/Models/PBR/BoxUnlit/BoxUnlit-processed.gltf

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
Why is this file needed? Can we just modify the original BoxUnlit?

Reply:
Oh whoops this shouldn't have been committed. Accidental output when using `gltf-pipeline` incorrectly to generate a GLB :sweat_smile: I'll remove this.